### PR TITLE
Update OpenCL documentation (with bonus)

### PR DESCRIPTION
**Pull request by denydias**
Update OpenCL documentation as per @dlemstra request in [ImageMagick/ImageMagick#4646](https://github.com/ImageMagick/ImageMagick/issues/4646).

Bonus: there's also a small addition to `script/session.php` so one can use PHP's built-in webserver on a local development environment. This can be safely ignored if it doesn't fit.

PS: I'm not native English speaker. Feel free to adjust the wording as nedded.

### CODE REVIEW

**Comment by j12i** [comment]
I caught some additional typos and fixed grammar in one place.
@denydias 

[0001-some-tiny-typos-and-1-grammar.patch.txt](https://github.com/ImageMagick/Website/files/7807421/0001-some-tiny-typos-and-1-grammar.patch.txt)

**Comment by denydias** [comment]
Thanks, @j12i!

File: script/session.php [hunk 1]
```diff
@@ -66,6 +66,9 @@
   if ($_SERVER["SERVER_ADDR"] == "10.144.245.30") {
     $use_sts = false;
   }
+  if (preg_match('/^PHP (\d+\.)?(\d+\.)?(\*|\d+) Development Server$/', $_SERVER["SERVER_SOFTWARE"]) === 1) {
```
Comment (dlemstra) [inline script/session.php:69]: Why was this added? Can you revert this change?

**Review by dlemstra — COMMENTED** [review]
Thanks for helping out, only a minor comment.

File: script/session.php (hunk 1 above)
Comment (j12i) [reply script/session.php:69]: They said why
> so one can use PHP's built-in webserver on a local development environment. This can be safely ignored if it doesn't fit.

Comment (j12i) [reply script/session.php:69]:
```suggestion

```

Comment (dlemstra) [reply script/session.php:69]: I completely missed that. I would prefer this to be removed.

Comment (j12i) [reply script/session.php:69]: If you don't use this change, the whitespace changes in the file should also be reverted, so the commit needn't touch it.

Comment (denydias) [reply script/session.php:69]: @dlemstra sure! Just cherry-pick the desired changes and ignore the rest.

Or do you want me to create a new PR without the session code?

**Comment by denydias** [comment]
I bit my tongue. Forgot to add OCL env and operator parameter on init test command. Fixed that above.

File: script/session.php (hunk 1 above)
Comment (dlemstra) [reply script/session.php:69]: Merged it and removed that part.

Comment (denydias) [reply script/session.php:69]: No worries. It won't affect the updated OpenCL content in any way. Thank you so much for merging it, @dlemstra.

Comment (j12i) [reply script/session.php:69]: > Merged it and removed that part.

@dlemstra the session.php changes still landed in Website@689b0fc?

Comment (dlemstra) [reply script/session.php:69]: They did but I pushed a patch to remove it: 0c4751d3aea763b84c599b587a605889a1490c70
